main.js

```diff
@@ -1,12 +1,28 @@
 const quadratic = (a, b, c) => {
-    // complete the code here
+       // complete the code here
+       // Step 1: Calculate the Discriminant
+       const discriminant = b * b - 4 * a * c; 
+    
+       if (discriminant >= 0) { // Step 2: Check if roots exist
+           const x1 = (-b + Math.sqrt(discriminant)) / (2 * a);
+           const x2 = (-b - Math.sqrt(discriminant)) / (2 * a);
+           return x1, x2;
```

Comment:
You have to return string format the same as expected input on the guideline not just returning the result values.